components/SourceTag.vue

```diff
@@ -37,6 +46,7 @@ export default {
   margin-right: 3px;
   text-transform: uppercase;
   position: relative;
+  height: min-content;
```

Comment:
This property means that the a source-tag can be the child of an element with `display: block;` (ie. a header) without changing its height to fill all the vertical space available